docs/en/latest/building-apisix.md

```diff
@@ -235,7 +235,9 @@ The steps below show how to run the test cases for APISIX:
    TEST_NGINX_BINARY=/usr/local/bin/openresty prove -Itest-nginx/lib -r t
    ```
 
-5. Run the tests by running:
+5. Install [LuaJIT](https://luajit.org/install.html). macOS users can use `brew` package manager to install LuaJIT.
```

Comment:
The luajit is installed with openresty by default

Reply:
Oh, I will close this PR then, I don't remember what problem I faced while installing.